packages/formation-react/scripts/build.js

```diff
@@ -49,7 +49,7 @@ const fileNames = [].concat.apply(
   [],
   [
     glob.sync('./src/components/**/*.@(js|jsx)', {
-      ignore: ['./**/*.unit.spec.jsx', './**/*.stories.jsx'],
+      ignore: ['./**/*.unit.spec.@(js|jsx)', './**/*.stories.@(js|jsx)'],
```

Comment:
This will prevent the build from grabbing unit tests and stories with `.js` instead of `.jsx`.

Thanks for this, @timwright12! :raised_hands: